lib/game_maker.ex

```diff
@@ -145,10 +144,10 @@ defmodule GameMaker do
   end
 
   def valid_symbol?(symbol) do
-    if String.length(symbol) < 1 do
+    if String.length(String.trim(symbol)) < 1 do
```

Comment:
Can this conditional be simplified to `String.length(String.trim(symbol)) >= 1`?